types/three/examples/jsm/webxr/XREstimatedLight.d.ts

```diff
@@ -0,0 +1,32 @@
+import { DirectionalLight, Group, LightProbe, XRFrame, WebGLRenderer, Texture } from '../../../src/Three';
+
+export class SessionLightProbe<TLight extends THREE.Group = THREE.Group> {
+    xrLight: TLight;
+    renderer: WebGLRenderer;
+    lightProbe: unknown;
```

Comment:
I was going to, but if i'm honest, I don't use the XR stuff enough to move it over meaningfully without knowing if it will break something. I will add that type.